javascript/lib/arrow.ts

```diff
@@ -0,0 +1,184 @@
+import { flatbuffers } from 'flatbuffers';
+import { org } from './Arrow_generated';
+var arrow = org.apache.arrow;
+import { vectorFromField, Vector } from './types';
+
+export function loadVectors(buf) {
+    var fileLength = buf.length, bb, footerLengthOffset, footerLength,
```

Comment:
I could move both of these functions into an `arrow.file` module, and similar functions could be implemented in `arrow.stream` in the future. Or perhaps it should be `arrow.io.[file,stream]` to allow for other modules.